### PR TITLE
uat android fixes

### PR DESCRIPTION
**Pull request by Salman1019864**
UAT Android App fixes

### CODE REVIEW

File: src/components/adb-account-kyc-openning/index.tsx [hunk 1]
```diff
@@ -91,18 +93,21 @@ const styles = StyleSheet.create({
     fontFamily: fonts.regular,
     fontSize: 12,
     maxWidth: '90%',
+    marginLeft:6
   },
   noteContainer: {
     backgroundColor: '#F5F5F5',
```
Comment (sharmal33) [inline src/components/adb-account-kyc-openning/index.tsx:99]: use theme colors

File: src/components/adb-account-opening-purpose/index.tsx [hunk 2]
```diff
@@ -178,7 +178,6 @@ const ADBAccountOpeningPurposeComponent = (
         const listItems = isSelectedMalaysia
           ? listBSData.filter((c) => c.value !== 'Malaysia')
```
Comment (sharmal33) [inline src/components/adb-account-opening-purpose/index.tsx:179]: why 'Malaysia' is hardcoded in here ?
get it from utils file

File: src/components/adb-account-opening-purpose/index.tsx [hunk 3]
```diff
@@ -437,7 +436,6 @@ const ADBAccountOpeningPurposeComponent = (
                     <View style={styles.taxInfomationsContainer}>
                       {listTaxes.map((t, index) => {
                         console.log('listTaxes -> item', t);
```
Comment (sharmal33) [inline src/components/adb-account-opening-purpose/index.tsx:438]: remove logs

File: src/components/adb-account-openning/index.tsx [hunk 4]
```diff
@@ -75,19 +75,13 @@ const ADBAccountOpeningComponent = (props: ADBAccountOpeningProps) => {
...
             in.
-            <TouchableOpacity>
-              <Text
-                style={styles.policyTextLinks}
-                onPress={handleOnPolicyPress}
-              >
-                {'Read more.'}
-              </Text>
-            </TouchableOpacity>
+            <Text style={styles.policyTextLinks} onPress = {handleOnPolicyPress}>{'Read more.'}</Text>
```
Comment (sharmal33) [inline src/components/adb-account-openning/index.tsx:83]: get it from translations file

File: src/components/adb-account-term-conditions/index.tsx [hunk 5]
```diff
@@ -58,11 +58,18 @@ const ADBAccountTermConditionsComponent = (
     setDocumentTitle('Privacy and policy');
     setIsShowDocument(true);
   };
+  const onViewProductDisclosure = () => {
+    const privacyLink = remoteConfig().getString('privacyPolicyTemplate');
+    console.log('onViewPrivacyAndPolicy => privacyLink', privacyLink);
```
Comment (sharmal33) [inline src/components/adb-account-term-conditions/index.tsx:63]: remove logs

File: src/components/adb-account-term-conditions/index.tsx [hunk 6]
```diff
@@ -58,11 +58,18 @@ const ADBAccountTermConditionsComponent = (
...
+    setDocumentLink(privacyLink);
+    setDocumentTitle('Product disclosure');
+    setIsShowDocument(true);
+  };
 
   const onViewPIDM = () => {
     const privacyLink = remoteConfig().getString('privacyPolicyTemplate');
     setDocumentLink(privacyLink);
-    setDocumentTitle('Privacy and policy');
+    setDocumentTitle(`PIDM's brochure`);
```
Comment (sharmal33) [inline src/components/adb-account-term-conditions/index.tsx:72]: get it from translation file

File: src/components/adb-account-term-conditions/index.tsx [hunk 7]
```diff
@@ -107,7 +114,7 @@ const ADBAccountTermConditionsComponent = (
           <View style={styles.right}>
             <Text style={styles.itemTitle}>
               I have an existing internet banking account with another bank in
-              Malaysia.
+              Malaysia in the same name (only applicable for Malaysian citizens)
```
Comment (sharmal33) [inline src/components/adb-account-term-conditions/index.tsx:117]: get it from translation file

File: src/components/adb-account-term-conditions/index.tsx [hunk 8]
```diff
@@ -124,12 +131,7 @@ const ADBAccountTermConditionsComponent = (
             <Text>{'•'}</Text>
             <Text style={styles.note}>
               {`I understand that this account is protected by PIDM up to RM250,000 for each depositor and I have received a copy of `}
```
Comment (sharmal33) [inline src/components/adb-account-term-conditions/index.tsx:133]: get all static string from translations file

File: src/components/adb-account-term-conditions/index.tsx [hunk 9]
```diff
@@ -138,25 +140,13 @@ const ADBAccountTermConditionsComponent = (
             <Text>{'•'}</Text>
             <Text style={styles.note}>
               {`By continuing, I agree and comply with {ADB}’s \nand AEON membership’s`}
```
Comment (sharmal33) [inline src/components/adb-account-term-conditions/index.tsx:142]: get this from translation file

File: src/components/adb-employment-details/index.tsx [hunk 10]
```diff
@@ -68,6 +68,7 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
       (d) => d.name === name
     );
     const selectedData = metadata?.fields[indexReligion];
+    console.log('selectedData',metadata?.fields,name)
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:71]: remove log

File: src/components/adb-employment-details/index.tsx [hunk 11]
```diff
@@ -165,6 +166,7 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
     setIsShowBottomSheet(true);
     try {
       const response = await onboardingService.getOccupationList();
+      console.log('getOccupationList',response)
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:169]: remove log

File: src/components/adb-employment-details/index.tsx [hunk 12]
```diff
@@ -179,8 +181,11 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
           searchBoxPlaceholder: 'Search',
           type: 'OptionsList',
         });
+        
       }
-    } catch {
+    } catch (error) {
+      console.log('error12',error)
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:187]: remove log

File: src/components/adb-employment-details/index.tsx [hunk 13]
```diff
@@ -232,6 +237,8 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
       ? bsData.items.filter((i: BSOption) => i.value.includes(searchText))
       : bsData.items
     : [];
+ 
+  console.log('bsData',bsData)
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:241]: remove log

File: src/components/adb-employment-details/index.tsx [hunk 14]
```diff
@@ -295,7 +302,7 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
             setFieldValue('annualIncome', currencyFormated);
           }
 
-          console.log('errors', errors);
+          console.log('errors1', errors);
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:305]: remove log

File: src/components/adb-employment-details/index.tsx [hunk 15]
```diff
@@ -416,7 +423,7 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
             </>
           )} */}
           <ADBInputField
-            prefixText={'MYR'}
+            prefixText={'RM'}
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:426]: get it from translation file

File: src/components/adb-employment-details/index.tsx [hunk 16]
```diff
@@ -442,7 +449,7 @@ const ADBPersonalDetailsComponent = (prop: ADBPersonalDetailsProps) => {
               {bsData && (
                 <ADBBottomSheet
                   items={displayData}
-                  title={bsData.title}
+                  title={bsData.name  === 'employmentSector'? "Employment Sector" : bsData.title}
```
Comment (sharmal33) [inline src/components/adb-employment-details/index.tsx:452]: get it from translation file

File: src/components/adb-enable-push-notification/index.tsx [hunk 17]
```diff
@@ -53,16 +53,17 @@ const styles = StyleSheet.create({
     height: 10,
   },
   noteSection: {
-    marginTop: 20,
+    marginTop: Platform.OS === 'ios'  ? 20 :30,
     marginBottom: 15,
     backgroundColor: '#F5F5F5',
```
Comment (sharmal33) [inline src/components/adb-enable-push-notification/index.tsx:58]: plz use theme colors

File: src/components/adb-enable-push-notification/index.tsx [hunk 18]
```diff
@@ -53,16 +53,17 @@ const styles = StyleSheet.create({
...
     borderRadius: 5,
     padding: 15,
+    
   },
   imageContent: {
     borderRadius: 30,
     backgroundColor: '#D9D9D9',
```
Comment (sharmal33) [inline src/components/adb-enable-push-notification/index.tsx:65]: use theme colors

File: src/components/adb-personal-details-review/index.tsx [hunk 19]
```diff
@@ -545,7 +545,7 @@ const ADBPersonalDetailsReviewComponent = (
   const displayData = bsData
     ? searchText && searchText.length > 0 && bsData.items?.length > 0
       ? bsData.items.filter((i: BSOption) => i.value.includes(searchText))
-      : bsData.items
+      : bsData.items.filter((i: BSOption) => i.value !== "Not Applicable")
```
Comment (sharmal33) [inline src/components/adb-personal-details-review/index.tsx:548]: get it from translation file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

**Review by usamabutt101 — APPROVED** [review]

**Pull request merged**